File: JAVA_COURSE/KOTLIN/TestDO/src/testdo/Test1.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package testdo;

/**
 *
 * @author LV
 */
public class Test1 {
    public int x;
    
   @Override
   protected void finalize() {
     // super.finalize();
    System.out.println(" Я умер " + x);
}
    
    
}

File: JAVA_COURSE/KOTLIN/TestDO/src/kotlintestdo/Test1.kt
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package kotlintestdo

/**
 *
 * @author LV
 */
class Test1 {
    var x: Int = 0

    protected fun finalize() {
        // super.finalize();
        println(" Я умер $x")
    }


}
